contract/src/deposit.rs

```diff
@@ -114,4 +164,39 @@ impl AuroraLaunchpadContract {
         );
         matches!(&self.config.deposit_token, DepositToken::Nep245((account_id, token_id)) if account_id == predecessor_account_id && token_id == &token_ids[0])
     }
+
+    fn create_refund_promise(&self, account: &IntentsAccount, amount: U128) -> Promise {
+        match &self.config.deposit_token {
+            DepositToken::Nep141(token_id) => ext_ft::ext(token_id.clone())
+                .with_attached_deposit(ONE_YOCTO)
+                .with_static_gas(GAS_FOR_FT_TRANSFER_CALL)
+                .ft_transfer_call(
+                    self.config.intents_account_id.clone(),
+                    amount,
+                    account.to_string(),
```

Comment:
For consistency, maybe it makes sense to use the intents `DepositMessage` struct to create the message instead of writing the account directly. Using `DepositMessage` from the start is more resilient to potential changes in the message format intents uses.